style.css

```diff
@@ -0,0 +1,79 @@
+
+.title {
+	font-size: 10px;
+}
+
+body{ 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "{" of a multi-line block (block-opening-brace-newline-after)](https://app.codacy.com/gh/WebHeroSchool/ad/pullRequest?prid=5498083)